src/config/gce_zone_network_health.yml

```diff
@@ -15,10 +15,19 @@ gcp:
   dimensions:
   - value: resource.labels.project_id
     id: project_id
+    compatibilityModeOnly: true
+  - value: resource.labels.project_id
+    id: gcp.project.id
   - value: resource.labels.zone
     id: zone
+    compatibilityModeOnly: true
+  - value: resource.labels.zone
+    id: gcp.region
   - value: resource.labels.region
     id: region
+    compatibilityModeOnly: true
+  - value: resource.labels.region
+    id: gcp.region
```

Comment:
TODO BŁ duplicate, keep only one as gcp.region - zone, because it's more detailed?